services/supervisord/supervisord.go

```diff
@@ -579,6 +579,9 @@ command =
 		--search.disableCache={{ .VMDBCacheDisable }}
 		--search.maxQueryLen=72KB
 		--search.latencyOffset=5s
+		--search.maxConcurrentRequests=32
+		--search.maxQueueDuration=20s
+		--search.maxQueryDuration=30s
```

Comment:
Looks like 30s is a default value:

https://hub.docker.com/r/victoriametrics/victoria-metrics/

> The maximum duration for search query execution (default 30s)

Reply:
Oh yeah, I forget increase search.maxQueryDuration. Thanks will fix it.